File: Lab_1/src/Square.java
public class Square extends Figure{
    private final int x, y, a;

    public Square(int x, int y, int a) {
        this.x = x;
        this.y = y;
        this.a = a;
    }

    @Override
    public void draw() {
        System.out.printf("Square(x = %d, y = %d, a = %d)\n", x, y, a);
    }
}

File: Lab_1/Square.cs
public class Square : Figure
{
    double x, y, a;
    public Square(double x, double y, double a)
    {
        this.x = x;
        this.y = y;
        this.a = a;
    }

    public override void drawFigure()
    {
        Console.WriteLine($"Square({x}, {y}, {a})");
    }
}